### ISSUE TITLE
Important notes displayed twice in *spacemacs* buffer.

### ISSUE THREAD

**Issue by laynor** [syl20bnr/spacemacs#12574]
#### Description :octocat:
Important notes displayed twice in *spacemacs* buffer.

#### Reproduction guide :beetle:
- Start Emacs

*Observed behaviour:* :eyes: :broken_heart:

The *spacemacs* buffer contains (cut for clarity):

```
      ╭─ Important Notes (Release 0.300.x) ────────────────────────────────────────────────╮
      │                                                                                    │
      │         ╭───────────────────────────────────────────────────────────╮              │
      │                                                                                    │
      │                         VERSION 0.300 IS ALMOST OUT!                               │
      │                                                                                    │
      │         ╰───────────────────────────────────────────────────────────╯              │
      │                                                                                    │
      │  The long awaited feature to have stable ELPA packages is now available in the     │
   8<----------------------------------------------------------------------------------------
      ╰─ Update your dotfile (SPC f e D) and packages after every update ──────────────────╯
      : 
      ╭─ Important Notes (Release 0.300.x) ────────────────────────────────────────────────╮
      │                                                                                    │
      │         ╭───────────────────────────────────────────────────────────╮              │
      │                                                                                    │
      │                         VERSION 0.300 IS ALMOST OUT!                               │
      │                                                                                    │
      │         ╰───────────────────────────────────────────────────────────╯              │
      │                                                                                    │
      │  The long awaited feature to have stable ELPA packages is now available in the     │
   8<----------------------------------------------------------------------------------------
      ╰─ Update your dotfile (SPC f e D) and packages after every update ──────────────────╯
```
*Expected behaviour:* :heart: :smile:
Should only contain:
```
      ╭─ Important Notes (Release 0.300.x) ────────────────────────────────────────────────╮
      │                                                                                    │
      │         ╭───────────────────────────────────────────────────────────╮              │
      │                                                                                    │
      │                         VERSION 0.300 IS ALMOST OUT!                               │
      │                                                                                    │
      │         ╰───────────────────────────────────────────────────────────╯              │
      │                                                                                    │
      │  The long awaited feature to have stable ELPA packages is now available in the     │
   8<----------------------------------------------------------------------------------------
      ╰─ Update your dotfile (SPC f e D) and packages after every update ──────────────────╯
```
#### System Info :computer:
- OS: gnu/linux
- Emacs: 27.0.50
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 0e0319968)
- Graphic display: t
- Distribution: spacemacs
- Editing style: hybrid
- Completion: helm
- Layers:
```elisp
(yaml html javascript haskell factor emacs-lisp common-lisp racket dap
      (python :variables python-test-runner
              '(nose pytest)
              python-backend 'lsp)
      shell-scripts helm cmake
      (auto-completion
       (haskell :variables haskell-completion-backend 'intero))
      fsharp better-defaults lsp git syntax-checking
      (version-control :variables version-control-diff-tool 'diff-hl))
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG SOUND DBUS GSETTINGS GLIB NOTIFY INOTIFY LIBSELINUX GNUTLS LIBXML2 FREETYPE HARFBUZZ XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM MODULES THREADS XWIDGETS PDUMPER GMP

#### Additional info
my `~/.emacs.d` is a symlink.

**Comment by duianto** [comment]
Confirmed.

But only in Ubuntu with Emacs 27.0.50.

Windows 10 Version 1903 with both Emacs 26.2 and 27.0.50,
only shows one instance of: `Important Notes (Release 0.300.x)`

On the latest develop branch with updated packages in both OS's.

#### System Info :computer:
- OS: gnu/linux
- Emacs: 27.0.50
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 0e0319968)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(auto-completion emacs-lisp git helm html markdown multiple-cursors org spell-checking syntax-checking treemacs version-control)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG SOUND GPM DBUS GSETTINGS GLIB NOTIFY INOTIFY ACL LIBSELINUX GNUTLS LIBXML2 FREETYPE HARFBUZZ M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM MODULES THREADS XWIDGETS LIBSYSTEMD PDUMPER LCMS2 GMP

#### System Info :computer:
- OS: windows-nt
- Emacs: 27.0.50
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 0e0319968)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(autohotkey auto-completion c-c++ dap emacs-lisp
            (git :variables)
            helm
            (html :variables css-enable-lsp t)
            (javascript :variables javascript-backend 'lsp)
            lsp markdown multiple-cursors org python spell-checking syntax-checking treemacs version-control)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG SOUND NOTIFY W32NOTIFY ACL GNUTLS LIBXML2 HARFBUZZ ZLIB TOOLKIT_SCROLL_BARS THREADS PDUMPER LCMS2 GMP

**Comment by jr0cket** [comment]
@duianto also confirmed on Ubuntu 18.10, 
Emacs 27.0.50 `GNU Emacs 27.0.50 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.4) of 2019-07-19`  
Spacemacs `develop` commit `b097d9cf0`

However, with the same Spacemacs commit but Emacs build from 2019-04-11 and Ubuntu 19.04 I do not get a duplicated message.  This does have a more recent install of spacemacs (sometime in the last 6 months) compared to the 18.10 machine which had Spacemacs first installed several years ago.

**Comment by duianto** [comment]
It doesn't seem to be related to the age of the Spacemacs installation, because I just installed a fresh copy of Spacemacs (develop) in Ubuntu 18.04.2:
`GNU Emacs 27.0.50 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.22.30) of 2019-07-26`

And I'm seeing duplicate release notes, I also noticed that there are duplicate instances of `Quick Help` (the `[?]` button in the spacemacs buffer).

Maybe it's a recent Emacs build issue.
Because there's only one instance in Windows with Emacs 26.2 and 27.0.50 (2019-06-08).

@laynor What's the output of `M-x emacs-version` for you?

**Comment by laynor** [comment]
@duianto I build emacs from sources every week.
```
GNU Emacs 27.0.50 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.1) of 2019-07-29
```

On my Arch linux machine I do not have this issue though, and I update emacs almost as frequently.

**Comment by duianto** [comment]
Could you check what the date of the Emacs version in Arch is. If it's got a recent date, then the Emacs build date might not be the source either.

**Comment by laynor** [comment]
Will do when I check home! Should be fairly recent though, maybe one or two weeks ago.

**Comment by duianto** [comment]
The issue seems to be occurring in the function:
https://github.com/syl20bnr/spacemacs/blob/b110ac09f94c6a412747172584c21baf3d31f507/core/core-spacemacs-buffer.el#L368

I added a message at the start of the function, and another one in the `while` loop.
the first message only runs once when the quick note is opened, but the `while` loop runs:
- `81` times when the note is duplicated (Ubuntu)
>SPACEMACS-BUFFER//NOTES-INSERT-NOTE
          (while (not (eobp)) [81 times]
- `41` times when the note is shown once (Windows):
>SPACEMACS-BUFFER//NOTES-INSERT-NOTE
          (while (not (eobp)) [41 times]

**Comment by rcoacci** [comment]
This seems related to dot-spacemacs. I got it fixed after creating a new .spacemacs and copying over my settings.

**Comment by howdoicomputer** [comment]
Is there a way of disabling that GIGANTIC message entirely?

**Comment by rationalis** [comment]
Still getting this as of e0aee1554454aff8183076b442b2903e5fa072c1 with 28.0.50 snapshot on Windows. Haven't tried redoing my .spacemacs completely (although I did update it manually after pulling develop) but it seems like that's probably the easiest way to figure out what's wrong...

**Comment by 3n-k1** [comment]
can confirm i am still getting this on fedora 32 x64, spacemacs' develop branch, commit 15d9391, emacs versions 27.1 (fedora repos) and 28.0.50 (current emacs git head, commit 6a5f9700846551a7f3795e257356dbab865116f4)

looks to me like the issue is somewhere after `spacemacs-buffer//notes-render-framed-text` ([link](https://github.com/syl20bnr/spacemacs/blob/develop/core/core-spacemacs-buffer.el#L288)), specifically somewhere after line 340 (i think). that was as close as i could figure things out

**Comment by duianto** [comment]
This line seems to be the source:
https://github.com/syl20bnr/spacemacs/blob/cbeafda39b3a547d1d9e86c08c5b186c006e1779/core/core-spacemacs-buffer.el#L385

because the quick help instructions aren't inserted when it's commented out.

In Emacs 27.1 after that line is evaluated, the variable `spacemacs-buffer--note-widgets` contains:
```
((text :args nil :value 
╭─ Quick Help ───────────────────────────────────────────────────────────────────────╮
│                                                                                    │
│  Press [TAB] or [J] to move to next button.                                        │
│                                                                                    │
│  Press [Shift-TAB] or [K] to move to previous button.                              │
│                                                                                    │
│  Press [c] to jump to org-agenda list if it is enabled.                            │
│                                                                                    │
│  Press [b] to jump to bookmark list if it is enabled.                              │
│                                                                                    │
│  Press [d] to jump to org-todo list if it is enabled.                              │
│                                                                                    │
│  Press [e] to jump to error list.                                                  │
│                                                                                    │
│  Press [m] to jump to the top menu.                                                │
│                                                                                    │
│  Press [o] to jump to any link or button.                                          │
│                                                                                    │
│  Press [p] to jump to project list if it is enabled.                               │
│                                                                                    │
│  Press [r] to jump to recent file list if it is enabled.                           │
│                                                                                    │
│  Press [w] to jump to warning list.                                                │
│                                                                                    │
│  Press [SPC] (in Vim editing style) or [Alt-m] (in Emacs editing style) to access  │
│  Spacemacs commands (will list only the Vim style ones for the rest of this        │
│  help).                                                                            │
│                                                                                    │
│  Press [SPC f e d] (or [Alt-m f e d] in Emacs style) to access .spacemacs file.    │
│                                                                                    │
│  Press [SPC h SPC] to access a list of documentation and supported layers. You     │
│  can type anything to narrow to a specific feature i.e. "python" for Python        │
│  layer. There are multiple lists and you can press [Ctrl-o] to switch between      │
│  them.                                                                             │
│                                                                                    │
│  When reading a document, you can use [SPC s j] to jump to a heading.              │
│                                                                                    │
│  Please consult Spacemacs documentation from [SPC h SPC] for more details.         │
│                                                                                    │
╰────────────────────────────────────────────────────────────────────────────────────╯ :sample-overlay #<overlay from 627 to 4107 in *spacemacs*> :field-overlay (#<marker (moves after insertion) at 4109 in *spacemacs*> . #<marker at 7589 in *spacemacs*>) :from #<marker (moves after insertion) at 627 in *spacemacs*> :to #<marker at 7590 in *spacemacs*>))
```

In Emacs 23.1, where the spacemacs buffer notes only appear once.
The variable contains almost the same thing, except for the last line
```
╰────────────────────────────────────────────────────────────────────────────────────╯ :field-overlay (#<marker (moves after insertion) at 627 in *spacemacs*> . #<marker at 4107 in *spacemacs*>) :from #<marker (moves after insertion) at 627 in *spacemacs*> :to #<marker at 4108 in *spacemacs*>))
```

The following property only appears in Emacs 27.1:
```
:sample-overlay #<overlay from 627 to 4107 in *spacemacs*>
```

<details><summary>
Windows 10, System Info (Click to expand)
</summary>
<pre>
#### System Info :computer:
- OS: windows-nt
- Emacs: 27.1
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. cbeafda39)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(autohotkey command-log emacs-lisp git haskell helm
            (javascript :variables javascript-backend 'lsp)
            lsp multiple-cursors org
            (python :variables python-backend 'lsp python-lsp-server 'mspyls)
            (shell :variables shell-default-position 'right)
            version-control
            (treemacs :variables treemacs-lock-width t))
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG SOUND NOTIFY W32NOTIFY ACL GNUTLS LIBXML2 HARFBUZZ ZLIB TOOLKIT_SCROLL_BARS MODULES THREADS JSON PDUMPER LCMS2 GMP

</pre>
</details>

**Comment by duianto** [comment]
The following PRs changes have been applied to the `develop` branch:
Fix display of notes in home buffer #14008
